packages/@react-spectrum/dnd/docs/dragAndDrop.mdx

```diff
@@ -40,6 +40,11 @@ selected rows and hitting <Keyboard>Enter</Keyboard>/<Keyboard>Space</Keyboard>.
 drop targets. Hitting <Keyboard>Enter</Keyboard> will then confirm the drop operation on the focused drop target. For screen readers, please follow the instructions announced
 when focusing the row's drag handle to begin a drag operation. Go ahead and try out drag and drop in the example below!
 
+
+## Drag and Drop Concepts
+
+TODO: add stuff from RFC
```

Comment:
?

Reply:
That looks like an outdated diff, try refreshing. I'm also doing a final edit sweep, so probably hold off till tmrw haha